cmd/kubeadm/app/images/images.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
+	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
```

Comment:
there is no need to add this import because `"k8s.io/kubernetes/cmd/kubeadm/app/constants"` is already included in the above line. :)